### PR TITLE
[docs] Add status header for humio exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for humio exporter.

### CODE REVIEW

File: exporter/humioexporter/README.md [hunk 1]
```diff
@@ -1,7 +1,12 @@
 # Humio Exporter
-Exports data to Humio using JSON over the HTTP [Ingest API](https://docs.humio.com/reference/api/ingest/).
 
-Supported pipeline types: traces (with metrics and logs to follow soon)
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | trace     |
```
Comment (TylerHelmuth) [inline exporter/humioexporter/README.md:6]:
```suggestion
| Supported pipeline types | traces    |
```

**Review by TylerHelmuth — APPROVED** [review]
/cc @xitric as code owner to verify beta stability

**Comment by Xitric** [comment]
Yes, I can verify that this exporter only has beta stability. To my knowledge, it has only been used by a small handful of people, including myself.

As for the failing link test, it seems that the address of the ingest API has changed. My browser does redirect correctly, however. I wonder if the test is more strict?

**Comment by djaglowski** [comment]
`check-links` failure appears to be related to the file modified by this PR, but unrelated to actual changes on this PR. If I'm not mistaken, this is working [as intended](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/9b58adfea64caa8fcc0074097c103e652f54598d/.github/workflows/check-links.yaml#L26), so should be addressed as part of this PR.

```
FILE: exporter/humioexporter/README.md

  [✓] https://github.com/open-telemetry/opentelemetry-collector#beta → Status: 200
  [✓] https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib → Status: 200
  [✓] https://docs.humio.com/reference/api/ingest/ → Status: 200
  ERROR: 2 dead links found!
  [✖] #Tagging → Status: 404
  [✖] #Inherited-Options → Status: 404
  [✓] https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md#tls-configuration-settings → Status: 200
  [✓] https://docs.humio.com/docs/ingesting-data/ingest-tokens/ → Status: 200
  [✓] testdata/config.yaml → Status: 200
  [✓] https://github.com/open-telemetry/opentelemetry-collector/tree/main/config/confighttp#client-configuration → Status: 200
  [✓] https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/exporterhelper/README.md#configuration → Status: 200
  [✓] https://docs.humio.com/docs/parsers/tagging/ → Status: 200
  [✓] https://docs.humio.com/reference/api/cluster-management-api/#setup-grouping-of-tags → Status: 200

  12 links checked.
  [✖] #Tagging → Status: 404
  [✖] #Inherited-Options → Status: 404
```

**Comment by Xitric** [comment]
Am I mistaken, or do those links work perfectly fine?

There seems to be a discussion on the markdown-link-check repository and some open issues stating that a regression on fragment anchors has been introduced recently. See tcort/markdown-link-check#91.

**Comment by TylerHelmuth** [comment]
I think core had this same issue: https://github.com/open-telemetry/opentelemetry-collector/issues/5118

**Comment by TylerHelmuth** [comment]
Core is able to get anchors to work tho: https://github.com/open-telemetry/opentelemetry-collector/runs/6492891036?check_suite_focus=true.

File: exporter/humioexporter/README.md [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
-Supported pipeline types: traces (with metrics and logs to follow soon)
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
+
+Exports data to Humio using JSON over the HTTP [Ingest API](https://docs.humio.com/reference/api/ingest/).
 
 > :construction: This exporter is currently intended for evaluation purposes only! It has yet to be enabled in the build.
```
Comment (TylerHelmuth) [inline exporter/humioexporter/README.md:11]: Should this be removed?  The exporter is in the contrib distribution

**Comment by TylerHelmuth** [comment]
@Frapschen @Xitric the anchor links need to be lowercase: https://github.com/TylerHelmuth/opentelemetry-collector-contrib/runs/6612328113?check_suite_focus=true

File: exporter/humioexporter/README.md (hunk 2 above)
Comment (Xitric) [reply exporter/humioexporter/README.md:11]: Yes, we forgot to remove that comment way back when the exporter was enabled. My former colleague and I have evaluated it internally (we are not affiliated with Humio), so we do consider it sufficiently stable for beta.

**Comment by TylerHelmuth** [comment]
> @TylerHelmuth Alright, that is good to know. I learned something new today. Is this something you or @Frapschen can handle? I am currently on vacation and only on my phone.

@Frapschen as the PR creator please update the anchor links

File: exporter/humioexporter/README.md (hunk 2 above)
Comment (Frapschen) [reply exporter/humioexporter/README.md:11]: ok ,I will remove the comment.

**Comment by Frapschen** [comment]
done:)

**Review by djaglowski — APPROVED** [review]
Thank you, @Frapschen

**Pull request merged**